src/mds/FSMapUser.h

```diff
@@ -54,6 +50,10 @@ class FSMapUser {
   void print_summary(Formatter *f, ostream *out);
 
   static void generate_test_instances(std::list<FSMapUser*>& ls);
+
+  std::map<fs_cluster_id_t, fs_info_t> filesystems;
+  fs_cluster_id_t legacy_client_fscid;
```

Comment:
This can also be initialized to FS_CLUSTER_ID_NONE.

Reply:
Done.